src/seedu/addressbook/Main.java

```diff
@@ -124,6 +132,17 @@ private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorage
         boolean isStorageFileSpecifiedByUser = launchArgs.length > 0;
         return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
     }
-
-
+    
+    /**
+     * Creates new StorageFile object based on the default storage path.
+     */
+    private void createNewFile() {
+    	 try {
+    		 storage = new StorageFile();
+    	     addressBook = storage.load();
```

Comment:
Take note of the indentation.
Line 141 is indented with a tab, do be careful and check your indentation.